include/qpid/dispatch/container.h

```diff
@@ -228,7 +228,7 @@ void qd_link_close(qd_link_t *link);
 void qd_link_detach(qd_link_t *link);
 void qd_link_free(qd_link_t *link);
 void *qd_link_get_node_context(const qd_link_t *link);
-void qd_link_restart_rx(qd_link_t *link);
+void qd_link_q2_restart_recv(void *context);
```

Comment:
Can we please call this qd_link_q2_restart_receive()

Reply:
Done